src/jupyter_contrib_nbextensions/nbextensions/notify/notify.js

```diff
@@ -8,9 +8,18 @@ Add this file to $(ipython locate)/nbextensions/
 
 */
 
-define(["require"], function (require) {
+define([
+  "jquery",
+  "base/js/namespace",
+  "require",
+], function ($, IPython, require) {
   "use strict";
 
+  var params = {
+    sticky: false,
+    sound: false
```

Comment:
You could support custom sounds by adding a url as a parameter. Maybe a name like `play_sound` might be better in case this gets added in future?